Quick start edits (#309)
solidjs/solid-docs-next: content/quick-start.mdx

# Quick Start Guide

## Solid playgrounds

> Experience Solid in your browser with our interactive playground throug visiting [interactive playground](https://playground.solidjs.com/).

Reviewer: ```suggestion
Experience Solid in your browser by visiting our [interactive playground](https://playground.solidjs.com/).
```

This will start the development server. 
Now, open your browser and navigate to `localhost:3000` to see your application running.

> #### Explore more templates

Reviewer 2: I think this better fits an aside since it's kinda like extra info on templates and where to find them

Author: asides will be used as callouts for additional content that are tangentially related vs added content. 

they should also be used sparingly - in this instance it's better to stay as a heading vs aside!

Solid is built on top of [Vite](https://vitejs.dev/), which offers a convenient way to handle environment variables. The following steps will show how to set up environment varaibles in a Solid project:

1. In the root directory of the project, create a file called `.env`. 
This file will store environment variables in the `key = value` format.

<Aside>
**Note:** To prevent accidental exposure of environment variables to the client, only variables prefixed with `VITE_` will be exposed.

For example:
```
VITE_SECRET_KEY = 123hello
DB_PASSWORD = foobar
```

Only the `VITE_SECRET_KEY` will be exposed to client source code, while `DB_PASSWORD` will not, as shown below.

```js
console.log(import.meta.env.VITE_SECRET_KEY) // 123hello
console.log(import.meta.env.DB_PASSWORD) // undefined
```
</Aside>
    
2. In your Solid code, you can access environment variables using `import.meta.env`. 
Vite will automatically inject the environment variables into your application.
```javascript
function MyComponent() {
  return (
    <div>
      <h2>Component with environment variable used {import.meta.env.VITE_VARIABLE_NAME}</h2>
    </div>
  )
}
export default MyComponent;
```

> For more information on environment variables in Vite and enabling TypeScript intellisense for environment variables, please refer to the [Vite Documentation](https://vitejs.dev/guide/env-and-mode.html#env-files).

Reviewer 2: This would also do better in an Aside tag

Author: same as above
